src/main/java/it/gov/pagopa/payment/controller/QRCodePaymentControllerImpl.java

```diff
@@ -0,0 +1,25 @@
+package it.gov.pagopa.payment.controller;
+
+import it.gov.pagopa.payment.dto.qrcode.TransactionCreated;
+import it.gov.pagopa.payment.dto.qrcode.TransactionCreationRequest;
+import it.gov.pagopa.payment.service.QRCodePaymentService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class QRCodePaymentControllerImpl implements
+    QRCodePaymentController {
+
+  private final QRCodePaymentService qrCodePaymentService;
+
+  public QRCodePaymentControllerImpl(QRCodePaymentService qrCodePaymentService) {
+    this.qrCodePaymentService = qrCodePaymentService;
+  }
+
+  @Override
+  public ResponseEntity<TransactionCreated> createTransaction(TransactionCreationRequest trxCreationRequest) {
+    TransactionCreated response = qrCodePaymentService.createTransaction(trxCreationRequest);
+    return new ResponseEntity<>(response, HttpStatus.CREATED);
```

Comment:
you could simplify this
```suggestion
  @ResponseStatus(code= HttpStatus.CREATED) // this annotation could be setted in the interface together with the request mapping annotation
  @Override
  public TransactionCreated createTransaction(TransactionCreationRequest trxCreationRequest) {
     return qrCodePaymentService.createTransaction(trxCreationRequest);
```